package.json

```
@@ -57,24 +57,24 @@
     "@polkadot/util-crypto": "^10.0.2",
     "@substrate/calc": "^0.2.8",
     "argparse": "^2.0.1",
-    "confmgr": "1.0.7",
-    "express": "^4.17.1",
-    "express-winston": "^4.1.0",
-    "http-errors": "^1.8.1",
-    "lru-cache": "^6.0.0",
-    "rxjs": "^7.5.5",
-    "winston": "^3.3.3"
+    "confmgr": "1.0.9",
+    "express": "^4.18.1",
+    "express-winston": "^4.2.0",
+    "http-errors": "^2.0.0",
+    "lru-cache": "^7.13.1",
+    "rxjs": "^7.5.6",
+    "winston": "^3.8.1"
   },
   "devDependencies": {
     "@substrate/dev": "^0.6.3",
     "@types/argparse": "2.0.10",
     "@types/express": "4.17.13",
-    "@types/express-serve-static-core": "4.17.25",
-    "@types/http-errors": "1.8.1",
-    "@types/lru-cache": "^5.1.1",
+    "@types/express-serve-static-core": "4.17.29",
```

Latest version [update](https://www.npmjs.com/package/@types/express-serve-static-core) that was published 6h ago is `4.17.30` : 
```suggestion
    "@types/express-serve-static-core": "4.17.30",
```